mewarpx/tests/test_capacitive_discharge.py

```diff
@@ -0,0 +1,366 @@
+        background_density=N_INERT,
+        background_temperature=T_INERT,
+        scattering_processes={
+            'elastic' : {
+                'cross_section' : cross_sec_direc+'ion_scattering.dat'
+            },
+            'back' : {
+                'cross_section' : cross_sec_direc+'ion_back_scatter.dat'
+            },
+            # 'charge_exchange' : {
+            #    'cross_section' : cross_sec_direc+'charge_exchange.dat'
+            # }
+        }
+    )
+
+    # Specific numbers match older run for consistency
+    run = diode_setup.DiodeRun_V1(
+        CATHODE_TEMP=1473.15,
+        CATHODE_A=3.0e5,
+        CATHODE_PHI=2.11,
+        ANODE_TEMP=400,
+        ANODE_PHI=1.2,
+        V_ANODE_CATHODE=1.21,
+        D_CA=D_CA,
+        NPARTPERSTEP=200,
+        TOTAL_CROSSINGS=2.0,
+        DIAG_CROSSINGS=2.0,
+        J_TOLERANCE=0.001,
+        CFL_FACTOR=2.0,
+        OFFSET=1e-06,
+        FORCE_2D_SCRAPER=use_2d_scraper,
+        MERGING=True,
+        MERGING_DV=100000,
+        MERGING_PERPERIOD=20,
+        MERGING_DXFAC=2,
+        MERGING_XYFAC=10,
+        CHECK_CHARGE_CONSERVATION=False,
+        NX=nx,
+        NZ=nz,
+        DT=DT,
+        TOTAL_TIMESTEPS=TOTAL_TIME
+    )
+    # besides resultsinfo none of these are implemented in mewarpx
+    run.setup_run(
+        init_reflection=False,
+        init_merging=False,
+        init_traceparticles=False,
+        init_runinfo=False,
+        init_fluxdiag=False,
+        init_resultsinfo=True
+    )
+
+    sim = run.init_simulation(warpx_collisions=[mcc_electrons, mcc_ions])
+
+    run.add_species_to_sim(electrons, particle_per_cell=number_per_cell_each_dim)
+    run.add_species_to_sim(ions, particle_per_cell=number_per_cell_each_dim)
+
+    run.init_warpx(sim=sim)
+    # if use_2d_scraper:
+    #     runtools.PHistDiag(
+    #         quantity_spec=[
+    #             ('uxbirth', -run.setupinfo.vmax/4., run.setupinfo.vmax/4.),
+    #             ('uzbirth', 0, run.setupinfo.vmax/4.),
+    #         ],
+    #         scraper=run.scraper,
+    #         diag_steps=run.diag_steps,
+    #         linres=30,
+    #         jslist=0,
+    #     )
+
+    #     runtools.PHistDiag(
+    #         quantity_spec=[
+    #             ('zold', warp.w3d.zmmin, warp.w3d.zmmax),
+    #         ],
+    #         scraper=run.scraper,
+    #         diag_steps=run.diag_steps,
+    #         linres=50,
+    #         jslist=0,
+    #         name='particle_histogramcurrent'
+    #     )
+
+    # warputil.warp_generate()
+
+    # # Run the main WARP loop
+    # while not run.runresults.terminate_flag:
+    #     warp.step(run.diag_steps)
+
+    #######################################################################
+    # Cleanup and final output                                            #
+    #######################################################################
+
+    # run.runresults.finalize_save()
+
+    # out, _ = capsys.readouterr()
+
+    # filelist = [
```

Comment:
Everything from here down can probably be eliminated for now in this test function.